api-objects/Appointment.ts

```diff
@@ -0,0 +1,37 @@
+export class Appointment {
+    public appointmentid: number;
```

Comment:
I don't think appointment Id is something that is sent to the API ever because it's auto generated.

Reply:
This is what is received by the client from the Server. Appointment ID is very useful if we need to update notes or make other changes.